README.md

```diff
@@ -1,7 +1,7 @@
-[![GitHub release](https://img.shields.io/github/release/cybozu-go/topolvm.svg?maxAge=60)][releases]
-[![Main](https://github.com/cybozu-go/topolvm/workflows/Main/badge.svg)](https://github.com/cybozu-go/topolvm/actions)
-[![GoDoc](https://godoc.org/github.com/cybozu-go/topolvm?status.svg)][godoc]
-[![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/topolvm)](https://goreportcard.com/report/github.com/cybozu-go/topolvm)
+[![GitHub release](https://img.shields.io/github/v/release/topolvm/topolvm.svg?maxAge=60)][releases]
+[![Main](https://github.com/topolvm/topolvm/workflows/Main/badge.svg)](https://github.com/topolvm/topolvm/actions)
+[![GoDoc](https://godoc.org/github.com/topolvm/topolvm?status.svg)][godoc]
```

Comment:
```suggestion
[![Main](https://github.com/topolvm/topolvm/workflows/Main/badge.svg)](https://github.com/topolvm/topolvm/actions)
[![PkgGoDev](https://pkg.go.dev/badge/github.com/topolvm/topolvm?tab=overview)](https://pkg.go.dev/github.com/topolvm/topolvm?tab=overview)
```

Reply:
Thank you for your information. But it seems that `pkg.go.dev` is not yet reflected this transfer.

So after this PR is merged and added this new package to `pkg.go.dev`, I will update this URL correctly.
ref: https://go.dev/about#adding-a-package

Reply:
Never mind.  Once this PR, including this snippet, is merged, the link will work
because `go.mod` will be updated at the same time.